app/views/helpers/category/update.phtml

```diff
@@ -243,5 +245,18 @@
 				</div>
 			</div>
 		</fieldset>
+
+		<fieldset>
+			<legend><?= _t('sub.import_export.export') ?></legend>
+			<div class="form-group">
+				<label class="group-name"><?= _t('sub.feed.export-as-opml.label') ?></label>
+				<div class="group-controls">
+					<a class="btn btn-important" href="<?= _url('index', 'opml', 'get', 'c_'.$this->category->id()) ?>" download="<?= 'f_'.$this->category->id()?> <?= $this->category->name() ?>.opml.xml">
```

Comment:
```suggestion
					<a class="btn btn-important" href="<?= _url('index', 'opml', 'get', 'c_'.$this->category->id()) ?>" download="f_<?= $this->category->id() ?> <?= $this->category->name() ?>.opml.xml">
```

Minor consistency

Reply:
one more thing. it is `c_` instead of `f_`